development/build/index.js

```diff
@@ -152,6 +152,7 @@ async function defineAndRunBuildTasks() {
       'getSelection',
       // globals `opera` needs to function
       'opr',
+      'Iterator',
```

Comment:
This needs an explanation too. Whatever new code needed it, it's running outside of lavamoat.